src/Sulu/Bundle/WebsiteBundle/Resources/config/services.xml

```diff
@@ -292,6 +292,13 @@
             <argument type="collection"/>
         </service>
 
+        <service id="sulu_website.webspace_reference_store"
+                 class="Sulu\Bundle\WebsiteBundle\ReferenceStore\WebspaceReferenceStore">
+            <argument type="service" id="sulu_core.webspace.request_analyzer" on-invalid="null"/>
+
+            <tag name="sulu_website.reference_store" alias="webspace"/>
```

Comment:
can we use the constant for the `alias` value here?

Reply:
thats not possible